sdk/provisioning/Azure.Provisioning.ManagedIdentities/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
```

Comment:
What does this correspond to?  https://www.nuget.org/packages/Azure.ResourceManager.ManagedServiceIdentities?  This should probably move back into the main package as well since we want user assigned identities everywhere.

Reply:
Yes, this is for user assigned identities.